carstore/gateway_api.go

```diff
@@ -0,0 +1,61 @@
+package carstore
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/filecoin-project/dagstore/mount"
+	cid "github.com/ipfs/go-cid"
+)
+
+var (
+	defaultURL = "https://ipfs.io/api/v0/dag/export"
```

Comment:
nit:  go-ipfs 0.13+ allows you to replace legacy RPC API at  `/api/v0/dag/export` with implementation-agnostic  Gateway request `GET https://ipfs.io/ipfs/{cid}?format=car`   (or `GET https://ipfs.io/ipfs/{cid}` with  `Accept: application/vnd.ipld.car`)

This way you will be able to swap gateway endpoint and its  implementation in the future + benefit from planned [selector support](https://github.com/ipfs/go-ipfs/issues/8769).